src/navigation/stacks/HarborStacks/HarborStacks.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import { HarborTabParamsList } from '@appTypes/navigation/harbor';
-import { StakeHarborScreen } from '../../../screens/Harbor';
+import { StakeHarborScreen, WithdrawHarborScreen } from '@screens/Harbor';
+import { WithdrawRequests } from '@screens/Harbor/WithdrawRequests/WithdrawRequests';
```

Comment:
create index.ts file there, instead of  '@screens/Harbor/_**WithdrawRequests/WithdrawRequests**_' ->  '@screens/Harbor/**_WithdrawRequests_**'